cmake/Platform/Mac/info.plist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we put this file together with the O3DE_SDK tool?

Reply:
Will move